### ISSUE TITLE
Specifying Width/Height in %

### ISSUE THREAD

**Issue by gor181** [clauderic/react-infinite-calendar#14]
Hello,

I have encountered following issue: http://prntscr.com/be6h27
Seems that specifying width and height in % is not possible despite documentation saying so?

Component used as:

```
import Calendar from 'react-infinite-calendar';

<Calendar
  width={'100%'}
  display={display}
  theme={CALENDAR_THEME}
  selectedDate={selectedDate}
  onSelect={this.onDateSelectHandler}
/>
```
Some parts are not provided for brevity.

Thanks,

**Comment by gor181** [comment]
This can also be solved by using 

```
width={screen.width}
height={screen.height - 50}
```

Which works fine on mobile and desktop.
Maybe just to update docs ?

**Comment by clauderic** [comment]
Yeah, for certain you need to provide a number width. This is because the VirtualScroll needs to know how many months it needs to render on-screen. I'll need to update the docs to make it clearer. Percentage width *should* work, but there's currently a bug I need to fix with the year picker and percentage width. 

If you want, you can look into using something along the lines of https://github.com/bvaughn/react-virtualized/blob/master/docs/AutoSizer.md or roll your own solution

**Comment by gor181** [comment]
Thanks @clauderic

**Comment by clauderic** [comment]
Just published `1.1.15` which now properly supports providing width as percentage as per the docs
